src/shared/common/components/summary/actions/action-summary-interests.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const renderRow = ({ label, value }) => {
+    return (
+        <tr key={label} className='govuk-table__row'>
+            <th className='govuk-table__header padding-left--small govuk-!-width-one-third'>{label}</th>
+            <td className='govuk-table__cell'>{value}</td>
+        </tr>
+    );
+};
+
+renderRow.propTypes = {
+    label: PropTypes.string.isRequired,
+    value: PropTypes.string.isRequired
+};
+
+const ActionSummaryInterests = (props) => {
+
+    const interests = [ ...props.props.items ];
+
+    if (interests.length < 1) {
+        return false;
+    }
+
+    return (
+        <>
+            <h2 className='govuk-heading-m'>Interest</h2>
+            { interests.map((item, index) => (
+                <>
+                    <table className='govuk-table margin-left--small'>
+                        <caption className='govuk-table__caption margin-bottom--small'>Interest { index + 1 }</caption>
```

Comment:
Is this index + 1 bit of this used for anything or purely display purposes? If it's used as a reference for business reasons such as an identifier we should use a different method of determining the identifier as this in frontend component is potentially unstable. Happy to talk through for further context

Reply:
It's just for display purposes. It's not used as a reference for anything, we just went for this as it was an easy way to separate the interests in the summary tab

Reply:
I guess if it's just for display purposes does it add anything to the UX and therefore is it required? I'm just thinking it could be confusing for the user/users might assume it's an identifier and start using it as a reference elsewhere, for example "in interest 2 there is xyz...".

Might be worth running past interaction designer?

Reply:
I've removed the index + 1 as part of the lifting to a separate component, as I can see how it would be confusing and isn't consistent with appeals